tools/setup_generation.py

```diff
@@ -40,7 +40,7 @@
 PACKAGE_GROUP = [ "taipy.config", "taipy.core", "taipy.gui", "taipy.rest", "taipy.auth", "taipy.enterprise" ]
 
 # Assuming that this script is located in <taipy-doc>/tools
-tools_dir = os.path.dirname(__file__).replace("\\", "/")
+tools_dir = os.path.dirname(os.path.abspath(__file__)).replace("\\", "/")
```

Comment:
What was the problem with that?

Reply:
On my Ubuntu WSL, the `os.path.dirname(__file__)` actually return an empty string, which is weird. I also did some searching and found others with similar issue on other Linux distro.